hexrdgui/hexrd_config.py

```diff
@@ -1555,9 +1555,14 @@ def default_detector(self):
     def is_roi_instrument_config(self):
         for det in HexrdConfig().detectors.values():
             if det.get('pixels', {}).get('roi', {}).get('value'):
-                 return True
+                return True
         return False
 
+    def detector_roi_group(self, detector_name):
+        if self.is_roi_instrument_config:
+            det = self.detector(detector_name)
+            return det.get('group', {}).get('value')
```

Comment:
I think a detector "group" might be applicable in other settings other than ROI. For example, for a chunked instrument (which won't have ROI), we might still want to constrain distances between detectors in the same group for calibration. 

So we might want to just have `detector_group()` and remove the `self.is_roi_instrument_config` check.

Reply:
Makes sense!

Reply:
Just pushed the fix for the calls that I missed - should be all fixed now.

> and in this places where the group name is checked in the file name, we should only do that if it is roi as well.

So for something like the chunked instrument example you mentioned above it may have a group key but we'd expect that we wouldn't have fewer images than detectors?

Reply:
Right! It would have a group key, but the number of images will match the number of detectors.

Reply:
Got it!